### DISCUSSION TITLE
Partial Least Squares & PySR

### DISCUSSION THREAD

**Discussion by jordanparker-me** [MilesCranmer/PySR#66] [Ideas]
I wanted to ask if anyone has seen an implementation of Partial Least Squares and Symbolic Regression. They seem to be a great fit. This is especially true for small datasets with high dimensionality.  

I would love to hear back on this from someone.

**Comment by MilesCranmer** [comment]
Sorry not sure why I didn't see this post. For others looking at this discussion, you can check out the `full_objective` parameter in PySR or the `loss_function` in SymbolicRegression.jl - it lets you define an arbitrary objective function.
